### DISCUSSION TITLE
4.3.3

### DISCUSSION THREAD

**Discussion by ismcagdas** [abpframework/abp#9364] [Releases]
4.3.3 Release

<hr /><em>This discussion was created from the release <a href='https://github.com/abpframework/abp/releases/tag/4.3.3'>4.3.3</a>.</em>

**Comment by kfrancis** [comment]
Are there going to be release notes generally on these github releases?

**Reply by ismcagdas** [reply]
@kfrancis yes, we will enhance minor releases and include description.
